tests/requirements.txt

```diff
@@ -2,8 +2,9 @@ flexmock>=0.10.3
 ordereddict
 responses>=0.9.0,<0.10.8
 pypng
-pytest>=4.1.0
+pytest>=4.1.0,<5
```

Comment:
What is the issue with pytest 5? Note that Fedora 33 will be released with pytest 5.

Reply:
Some conflict between Coverage, pytest-html, and pytest.